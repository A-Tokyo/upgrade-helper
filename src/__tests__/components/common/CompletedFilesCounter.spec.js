import React from 'react'
import { render } from '@testing-library/react'
import CompletedFilesCounter from '../../../components/common/CompletedFilesCounter'

it('renders without crashing', () => {
  const { container } = render(<CompletedFilesCounter />)

  expect(container).toMatchSnapshot()
})
