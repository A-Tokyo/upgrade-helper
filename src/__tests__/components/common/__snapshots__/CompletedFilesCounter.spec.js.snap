// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <div
    class="sc-AykKC dtOdrK"
  >
    <span
      class="completedAmount"
    />
     /
    <div
      style="position: relative;"
    />
  </div>
</div>
`;
